volatility3/framework/layers/qemu.py

```diff
@@ -192,6 +263,45 @@ def _load_segments(self):
             else:
                 raise exceptions.LayerException(self._name, f'QEMU unknown section encountered: {section_byte}')
 
+    def _fallback_determine_architecture(self) -> str:
+        architecture_pattern = rb'pc-(i440fx|q35)-(\d{1,2}\.\d{1,2}|[\w\d\.]+)'
+        old_suffix = "-2.0"
```

Comment:
Rename to `default_suffix`, since it is actually the "New Default" for i440fx that we want to use as default fallback and not the "Old Default".

Reply:
Yep, good point.  5:)